.travis.yml

```diff
@@ -106,7 +152,7 @@ after_success:
   - mamba install anaconda-client
   - |
     # Only upload builds from tags
-    if [[ $TRAVIS_PULL_REQUEST == false && $TRAVIS_REPO_SLUG == "liulab-dfci/MAESTRO"
+    if [[ $TRAVIS_PULL_REQUEST == false && $TRAVIS_REPO_SLUG == "dongqingsun96/MAESTRO"
```

Comment:
This line should be changed back to point to `liulab-dfci/MAESTRO`

Reply:
Sure, I will change back to point to `liulab-dfci/MAESTRO`.